tests/stub/routing.py

```diff
@@ -142,7 +198,7 @@ def router_script_with_unknown_failure(self):
 
         C: HELLO {"scheme": "basic", "credentials": "c", "principal": "p", "user_agent": "007", "routing": #HELLO_ROUTINGCTX# #EXTRA_HELLO_PROPS#}
         S: SUCCESS {"server": "#SERVER_AGENT#", "connection_id": "bolt-123456789"}
-        C: ROUTE #ROUTINGCTX# [] "adb"
+        C: ROUTE #ROUTINGCTX# [] "*"
```

Comment:
Was there some need to support different db?

Reply:
Yup, `test_should_fail_discovery_when_router_fails_with_unknown_code` calls `verifyConnectivity` on that router. So in that case the DB is either `system` or `null`, depending on the driver's implementation of `verifyConnectivity`. Meanwhile, `test_should_request_rt_from_all_initial_routers_until_successful` uses that router script with a session on `adb`.